mypy_django_plugin/django/context.py

```diff
@@ -71,13 +72,20 @@ class LookupsAreUnsupported(Exception):
 
 
 class DjangoContext:
-    def __init__(self, django_settings_module: str) -> None:
-        self.django_settings_module = django_settings_module
+    def __init__(self, plugin_config: DjangoPluginConfig) -> None:
+        if isinstance(plugin_config, str):
+            raise RuntimeError("Old API is used: we now require DjangoPluginConfig instance")
```

Comment:
So people who have older djangorestframework-stubs installed and update django-stubs will receive this message?

Seems a bit cryptic. How about "Older djangorestframework-stubs versions are no longer supported, please update djangorestframework-stubs" or similar?

I doubt there are any other projects that import djagno-stubs internals, anyway.

Reply:
The only project is `drf-stubs`. I don't really like this solution now. See https://github.com/typeddjango/djangorestframework-stubs/pull/433

I propose this plan:
- I remove this error
- We release a new version of `drf-stubs` with the upper bound of `django-stubs` set to the current version of `django-stubs`
- We merge this and release a new version of `django-stubs`
- We merge https://github.com/typeddjango/djangorestframework-stubs/pull/433 and release a new version of `drf-stubs` with no upper bound

Reply:
On the other hand, it won't solve the problem of only upgrading `django-stubs` with old `drf-stubs` 😞 

Ideas?

Reply:
Ok, I was able to solve all the compat problems in Python code. The easiest way for some reason is always the hardest one to find.